debs/xenial/hashdeep/Makefile

```diff
@@ -0,0 +1,36 @@
+NAME          = hashdeep
+VERSION       ?= 4.4
+RELEASE       ?= 1~16.04
+DEB_TOPDIR    = "/debbuild"
+DOCKER_VOLUME = "/src"
+DOCKER_IMAGE  = "debbuild-$(NAME)-$(VERSION)"
+
+.PHONY: build-docker-image build deb-build deb-clean deb-test update-changelog
+
+all: build-docker-image build
+
+build-docker-image:
+	@echo "==> Building Docker image with build environment."
+	@docker build --tag "$(DOCKER_IMAGE)" --build-arg GIT_URL="$(GIT_URL)" --build-arg GIT_BRANCH=v"$(VERSION)" --build-arg PACKAGE="$(NAME)" .
```

Comment:
I've just realized that these build arguments that we're passing are also not being used in the Dockerfile.